services/huaweicloud/src/main/java/org/eclipse/osc/orchestrator/plugin/huaweicloud/builders/packer/Ocl2Packer.java

```diff
@@ -0,0 +1,79 @@
+package org.eclipse.osc.orchestrator.plugin.huaweicloud.builders.packer;
+
+import java.util.Optional;
+import lombok.extern.slf4j.Slf4j;
+import org.eclipse.osc.services.ocl.loader.Artifact;
+import org.eclipse.osc.services.ocl.loader.BaseImage;
+import org.eclipse.osc.services.ocl.loader.Ocl;
+import org.eclipse.osc.services.ocl.loader.Provisioner;
+
+@Slf4j
+class Ocl2Packer {
+    private final Ocl ocl;
+    private final Artifact artifact;
+
+    Ocl2Packer(Ocl ocl, Artifact artifact) {
```

Comment:
I think `ocl` is enogh for Ocl2Packer, how about unpacking the `artifact` from the `ocl` object?